doc/rst/quick.rst

```diff
@@ -129,8 +129,8 @@ The tools as delivered are to be used within a command line environment, though
 
 .. warning::
 
-    The firmware must be compiled using a specific version of the tools. Make sure that you download the correct version of the tools.
-    Older versions of tools are available from the `TOOLS ARCHIVE` section of http://www.xmos.com/software-tools
+    USB Audio applications are compiled using the `XCommon CMake <https://www.xmos.com/documentation/XM-014363-PC-10/html/tools-guide/tools-ref/xcommon_cmake.html#xcommon-cmake-build-system>`_ build system.
+    The minimum XTC tools version that supports XCommon CMake is 15.3.0. Ensure that the firmware is compiled using XTC Tools version 15.3.0 or above.
```

Comment:
Same comment on tools version. We only want it in one, maintained place (also means docs don't need updating every tools release)